ietf/api/tests.py

```diff
@@ -758,6 +766,7 @@ def do_rfc_with_broken_history_test(self, draft_name):
                 content_url=rfc.get_href(),
                 name=rfc.canonical_name(),
                 previous=f'{draft.name}-10',
+                previous_url= f'https://www.ietf.org/archive/id/{draft.name}-10.txt',
```

Comment:
It will save hassle later to pull that path out of settings instead of hardcoding it, even in the tests.

Reply:
Updated.